راهنمای_کداسپیس.md

```
@@ -0,0 +1,160 @@
+   - به ترمینالی که Django در آن اجرا می‌شود نگاه کنید
+
+3. **تأیید OTP و ثبت‌نام**:
+   - کد 6 رقمی را وارد کنید
+   - اطلاعات کاربری را تکمیل کنید
+
+4. **ورود به سیستم**:
+   - با موبایل و رمز عبور وارد شوید
+
+## 🧪 تست‌های API
+
+### اجرای تست‌های واحد:
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)